### DISCUSSION TITLE
cannot connect rsync server

### DISCUSSION THREAD

**Discussion by HorseBein** [DOI-USGS/ISIS3#5215] [Q&A]
hi, 
I have a trouble druing pulling the LRO and hayabusa spice data using rsync command on linux system today.  My colleagues also met the problem.  I followed FAQs in isis3 install-guide website (https://isis.astrogeology.usgs.gov/documents/LegacyInstallGuide/index.html),  type command: rsync isisdist.astrogeology.usgs.gov::    
or alternative commands. It still doesn't work. The error displayed is 

--rsync: failed to connect to isisdist.astrogeology.usgs.gov (184.164.152.106): Resource temporarily unavailable (11)
--rsync error: error in socket IO (code 10) at clientserver.c(127) [Receiver=3.1.3]

Pulling data using rsync in last October was okay, and no settings were modifed so far. I guess the rsync server might oversleep🌙. 

Has anyone encoutnered this problem recently?

**Comment by jlaura** [comment]
@HorseBein Thanks for the issue. Please see the [README](https://github.com/DOI-USGS/ISIS3/blob/dev/README.md#full-isis-data-download). As of November 30, 2022 the rsync server has been shutdown. Please update to use the `downloadIsisData` script.

**Comment by HorseBein** [comment]
@jlaura Thanks a lot!
